lib/QubitLftSyncer.class.php

```diff
@@ -92,8 +92,9 @@ private function repairEsChildrenLftValues()
         $results = QubitPdo::fetchAll($sql, $params, ['fetchMode' => PDO::FETCH_ASSOC]);
 
         $bulk = new Elastica\Bulk(QubitSearch::getInstance()->client);
-        $bulk->setIndex(QubitSearch::getInstance()->index->getName());
-        $bulk->setType('QubitInformationObject');
+        $type = 'QubitInformationObject';
+        $bulk->setIndex(QubitSearch::getInstance()->index->getIndex($type)->getName());
+        $bulk->setType(QubitSearch::getInstance()->index->getIndexName($type));
```

Comment:
Shouldn't this be constant? `$bulk->setType(QubitSearch::getInstance()::ES_TYPE);`